File: ASPNETCore/HowTo/WebApiWPFSample/WebApiWPFSample/src/Modules/Excel/Dtos/DeleteColumns.dto.cs
namespace WebApiConsoleSample
{
    public class DeleteColumnsDto : BaseDto
    {
        /**The excel file name that storage manager can recognize. */
        public string ExcelPath { set; get; }

        /**The sheet name.*/
        public string SheetName { set; get; }

        /**The column indexes. format: {num}; {num},{num},...; {num}-{num}. for example: 1,3-5 */
        public string ColumnIndexs { set; get; }

    }
}

File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/DeleteColumns.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class DeleteColumnsDto extends BaseDto {
    
    /**The excel file name that storage manager can recognize. */
    public ExcelPath: string;

    /**The sheet name.*/
    public SheetName: string;

    /**The column indexes. format: {num}; {num},{num},...; {num}-{num}. for example: 1,3-5 */
    public ColumnIndexs: string;

}